src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const carPrice = 40;
+  let result = 0;
+
+  if (days >= 7) {
+    result = days * carPrice - 50;
+
+    return result;
+  }
+
+  if (days >= 3) {
+    result = days * carPrice - 20;
+
+    return result;
+  } else {
+    return 80;
```

Comment:
The 'else' clause returns a fixed value of 80, which is incorrect. It should calculate the cost as 'days * carPrice' for cases where days are less than 3.

Reply:
done